### PR TITLE
Issue #468 bulk ops statistics

### PR DESCRIPTION
**Pull request by ljacomet**
* Added `OperationObserver` for getAll, putAll and removeAll
* Added counter for `getAll` miss
* Replace `AtomicLong` with `LongAdder` for bulk counters

### CODE REVIEW

File: core/src/main/java/org/ehcache/Ehcache.java [hunk 1]
```diff
@@ -91,13 +94,16 @@
...
+  private final OperationObserver<RemoveAllOutcome> removeAllObserver = operation(RemoveAllOutcome.class).named("removeAll").of(this).tag("cache").build();
   private final OperationObserver<ConditionalRemoveOutcome> conditionalRemoveObserver = operation(ConditionalRemoveOutcome.class).named("conditionalRemove").of(this).tag("cache").build();
   private final OperationObserver<CacheLoadingOutcome> cacheLoadingObserver = operation(CacheLoadingOutcome.class).named("cacheLoading").of(this).tag("cache").build();
   private final OperationObserver<PutIfAbsentOutcome> putIfAbsentObserver = operation(PutIfAbsentOutcome.class).named("putIfAbsent").of(this).tag("cache").build();
   private final OperationObserver<ReplaceOutcome> replaceObserver = operation(ReplaceOutcome.class).named("replace").of(this).tag("cache").build();  
-  private final ConcurrentMap<BulkOps, AtomicLong> bulkMethodEntries = new ConcurrentHashMap<BulkOps, AtomicLong>();
+  private final ConcurrentMap<BulkOps, LongAdder> bulkMethodEntries = new ConcurrentHashMap<BulkOps, LongAdder>();
```
Comment (chrisdennis) [inline core/src/main/java/org/ehcache/Ehcache.java]: EnumMap with initialized mappings to LongAdders?

File: core/src/main/java/org/ehcache/Ehcache.java [hunk 2]
```diff
@@ -1068,15 +1103,15 @@ private void checkNonNullContent(Collection<?> collectionOfThings) {
   }
 
   private void addBulkMethodEntriesCount(BulkOps op, long count) {
-    AtomicLong current = bulkMethodEntries.get(op);
+    LongAdder current = bulkMethodEntries.get(op);
     if (current == null) {
```
Comment (chrisdennis) [inline core/src/main/java/org/ehcache/Ehcache.java]: Once we have a pre-inited EnumMap we can do away with this null check.

File: core/src/main/java/org/ehcache/Ehcache.java (hunk 1 above)
Comment (ljacomet) [inline core/src/main/java/org/ehcache/Ehcache.java]: Yes!

File: core/src/main/java/org/ehcache/Ehcache.java (hunk 2 above)
Comment (ljacomet) [inline core/src/main/java/org/ehcache/Ehcache.java]: And others ... will change that

**Pull request merged**